include/seqan3/alphabet/union_alphabet.hpp

```diff
@@ -258,11 +308,36 @@ struct union_alphabet
         return value >= rhs.value;
     }
     //!@}
+
+protected:
+    //! \privatesection
+    // conversion tables
+
+    constexpr auto from_base_(const variant_type & alphabet_v) const {
```

Comment:
now however it is not clear, why some of the static functions are static functions declared inside the class and others are free functions in `seqan3::detail::union_alphabet`... maybe move this function to the private namespace, too?